airgun/views/host.py

```diff
@@ -297,3 +317,52 @@ class HostsChangeEnvironment(BaseLoggedInView):
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.title, exception=False) is not None
+
+
+class HostsAssignOrganization(BaseLoggedInView):
+    title = Text(
+        "//h4[text()='Assign Organization"
+        " - The following hosts are about to be changed']")
+    table = SatTable("//div[@class='modal-body']//table")
+    keep_selected = Checkbox(id='keep_selected')
+    organization = Select(id='organization_id')
+    on_mismatch = FormRadioGroup(
+        locator="//div[@class='modal-body']//div[@id='content']/form")
+    submit = Text('//button[@onclick="submit_modal_form()"]')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class HostsAssignLocation(BaseLoggedInView):
+    title = Text(
+        "//h4[text()='Assign Location"
+        " - The following hosts are about to be changed']")
+    table = SatTable("//div[@class='modal-body']//table")
+    keep_selected = Checkbox(id='keep_selected')
+    location = Select(id='location_id')
+    on_mismatch = FormRadioGroup(
+        locator="//div[@class='modal-body']//div[@id='content']/form")
+    submit = Text('//button[@onclick="submit_modal_form()"]')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class HostsAssignCompliancePolicy(BaseLoggedInView):
+    title = Text(
+        "//h4[text()='Assign Compliance Policy"
+        " - The following hosts are about to be changed']")
+    # table = SatTable("//div[@class='modal-body']//table")
```

Comment:
?

Reply:
commented to fast read that view :) , need to un-comment it